packages/server/test/support/helpers/e2e.coffee

```diff
@@ -35,12 +35,12 @@ pathUpToProjectName = Fixtures.projectPath("")
 
 DEFAULT_BROWSERS = ['electron', 'chrome']
 
-stackTraceLinesRe = /^(\s+)at\s(.+)/gm
+stackTraceLinesRe = /^(\s*).*?(@|at).*\.(js|coffee|ts|html)(-\d+)?:\d+:\d+[\n\S\s]*?(\n\s*\n)/gm
 browserNameVersionRe = /(Browser\:\s+)(Custom |)(Electron|Chrome|Canary|Chromium|Firefox)(\s\d+)(\s\(\w+\))?(\s+)/
 availableBrowsersRe = /(Available browsers found are: )(.+)/g
 
 replaceStackTraceLines = (str) ->
```

Comment:
we don't have any unit tests for functions like this?

Reply:
no, we're really just rolling the dice and assuming we don't make any mistakes with these. the fact that we have tons of snapshots already provides *some* assurances, but there's no explicit testing of `e2e.coffee`.

Reply:
I will add a comment on the regex so the intended behavior is at least written somewhere